managed-ledger/src/test/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerTest.java

```diff
@@ -3193,7 +3193,7 @@ public void avoidUseSameOpAddEntryBetweenDifferentLedger() throws Exception {
 
         @Cleanup("shutdown")
         ManagedLedgerFactoryImpl factory = new ManagedLedgerFactoryImpl(metadataStore, bkc, config);
-        ManagedLedgerImpl ledger = (ManagedLedgerImpl) factory.open("my_test_ledger");
+        ManagedLedgerImpl ledger = (ManagedLedgerImpl) factory.open("avoidUseSameOpAddEntryBetweenDifferentLedger");
```

Comment:
Seems that we have reinitialized the test env before running each test method. https://github.com/apache/pulsar/blob/master/managed-ledger/src/test/java/org/apache/bookkeeper/test/MockedBookKeeperTestCase.java#L69-L85

Why did they affect each other here?

Reply:
Oh, I missed this one, let me check the detailed logs.